tests/test_wave_03.py

```diff
@@ -54,13 +53,9 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert friends_unique_movies[0] != friends_unique_movies[1] != friends_unique_movies[2]    
```

Comment:
Great start! Although it's important to know if there are any duplicates in the `friends_unique_movies`, as noted in the title of the test, we should probably also confirm that the correct movies are in the list. If we add asserts to check if each of the three movies are in the list, we'll also know for sure there are no duplicates since none of those movies are the same.